Replace TryFrom with custom TryFromJsValue trait (#3709)
rustwasm/wasm-bindgen: CHANGELOG.md

* The DWARF section is now correctly modified instead of leaving it in a broken state.
  [#3483](https://github.com/rustwasm/wasm-bindgen/pull/3483)

> * Custom TryFromJsValue Trait: Fixed an issue where #[wasm_bindgen] automatically derived the TryFrom trait for any struct, preventing custom TryFrom<JsValue> implementations. The wasm-bindgen derive macro has been updated to utilize the new TryFromJsValue trait.    [#3709](https://github.com/rustwasm/wasm-bindgen/pull/3709)
> 

Reviewer: ```suggestion

* Fixed an issue where `#[wasm_bindgen]` automatically derived the `TryFrom` trait for any struct, preventing custom `TryFrom<JsValue>` implementations. It has been updated to utilize a new `TryFromJsValue` trait instead.
  [#3709](https://github.com/rustwasm/wasm-bindgen/pull/3709)
```
